urbanopt-scenario-gem.gemspec

```diff
@@ -21,19 +21,19 @@ Gem::Specification.new do |spec|
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
+  spec.required_ruby_version = '~> 2.2.4'
 
   spec.add_development_dependency 'bundler', '~> 1.14'
-  spec.add_development_dependency 'rake', '12.3.1'
-  spec.add_development_dependency 'rspec', '3.7.0'
-  spec.add_development_dependency 'github_api', '~> 0.18.0'
-  # Fix rack version temporarily to work with Ruby 2.2.4
-  spec.add_development_dependency 'rack', '2.1.2'
+  spec.add_development_dependency 'rake', '12.3'
+  spec.add_development_dependency 'rspec', '3.7'
+  spec.add_development_dependency 'github_api', '~> 0.18'  
 
   spec.add_dependency 'json-schema'
   spec.add_dependency 'json_pure'
-  spec.add_dependency 'openstudio-model-articulation', '~> 0.1.0'
-  spec.add_dependency 'urbanopt-core', '~> 0.1.0'
+  spec.add_dependency 'openstudio-model-articulation', '~> 0.1'
```

Comment:
we want the openstudio/urbanopt gems to stay super specified during active SDK development.  @nllong, is that right?

Reply:
Confirmed this with Nick